kayenta-sql/src/main/resources/db/changelog/202303091551-create-canary-archive-table.yml

```diff
@@ -0,0 +1,29 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-canary-archive-table
+      author: alexey.bedonik
+      changes:
+        - createTable:
+            tableName: canary_archive
```

Comment:
For this table (and the rest), it may be helpful to set some MySQL settings since it defaults to bullshit. You can use something like this in your create table statement:

```
    - modifySql:
        dbms: mysql
        append:
          value: " engine innodb DEFAULT CHARSET=utf8mb4 COLLATE utf8mb4_unicode_ci"
```